src/common/options.cc

```diff
@@ -2247,77 +2248,82 @@ std::vector<Option> get_global_options() {
 
     Option("osd_max_backfills", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(1)
-    .set_description(""),
+    .set_description("maximum number of concurrent backfills per OSD"),
 
     Option("osd_min_recovery_priority", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(0)
-    .set_description(""),
+    .set_description("minimum priority below which recovery is not performed")
+    .set_long_description("The purpose here is to prevent the cluster from performany *any* low priority work (e.g., rebalancing) below this threshold and focus solely on higher priority work (e.g., replicated degraded objects)."),
```

Comment:
s/performany/perform/

Reply:
"replicated degraded objects" does not sound right, how about "replicating degraded objects"?